src/plugins/OktaPluginA11y.ts

```diff
@@ -183,106 +9,47 @@ const isEqual = (a,b) => [...Object.keys(a)].every((k) => a[k] === b[k]);
-      setAttr('.mfa-onprem',                 { 'aria-label': 'Enroll Custom On-prem' });
-      setAttr('.mfa-rsa',                    { 'aria-label': 'Enroll RSA SecurID' });
-      setAttr('.mfa-symantec',               { 'aria-label': 'Enroll Symantec VIP' });
-      setAttr('.mfa-webauthn',               { 'aria-label': 'Enroll Security key or Biometric Authenticator' });
-      setAttr('.mfa-yubikey',                { 'aria-label': 'Enroll Yubikey' });
-
-      // set aria-label for buttons
-      setAttr('.authenticator-button[data-se="custom_otp"] a',        { 'aria-label': 'Enroll custom OTP' });
-      setAttr('.authenticator-button[data-se="duo"] a',               { 'aria-label': 'Enroll Duo Security' });
-      setAttr('.authenticator-button[data-se="external_idp"] a',      { 'aria-label': 'Enroll IDP Authenticator' });
-      setAttr('.authenticator-button[data-se="google_otp"] a',        { 'aria-label': 'Enroll Google Authenticator' });
-      setAttr('.authenticator-button[data-se="okta_email"] a',        { 'aria-label': 'Enroll email' });
-      setAttr('.authenticator-button[data-se="okta_password"] a',     { 'aria-label': 'Enroll password' });
-      setAttr('.authenticator-button[data-se="okta_verify-push"] a',  { 'aria-label': 'Enroll Okta Verify (Push Notification)' });
-      setAttr('.authenticator-button[data-se="okta_verify-totp"] a',  { 'aria-label': 'Enroll Okta Verify (Code)' });
-      setAttr('.authenticator-button[data-se="okta_verify"] a',       { 'aria-label': 'Enroll Okta Verify' });
-      setAttr('.authenticator-button[data-se="onprem_mfa-otp"] a',    { 'aria-label': 'Enroll custom on-prem authenticator' });
-      setAttr('.authenticator-button[data-se="phone_number"] a',      { 'aria-label': 'Enroll phone' });
-      setAttr('.authenticator-button[data-se="rsa_token-otp"] a',     { 'aria-label': 'Enroll RSA SecurID' });
-      setAttr('.authenticator-button[data-se="security_question"] a', { 'aria-label': 'Enroll security question' });
-      setAttr('.authenticator-button[data-se="symantec_vip"] a',      { 'aria-label': 'Enroll Symantec VIP' });
-      setAttr('.authenticator-button[data-se="webauthn"] a',          { 'aria-label': 'Enroll Security key or Biometric Authenticator' });
-      setAttr('.authenticator-button[data-se="yubikey_token"] a',     { 'aria-label': 'Enroll Yubikey' });
-    }
+    /**
+     * Hide authenticator icon from screen readers to prevent redundancy. The 
+     * sibling element .authenticator-description conveys the same information.
+     * 
+     * https://www.w3.org/WAI/tutorials/images/decision-tree/
+     * https://www.w3.org/WAI/tutorials/images/decorative/
+     */
+    document.querySelectorAll('.authenticator-list').forEach((list) => {
+      list.querySelectorAll('.authenticator-row')
+        .forEach((row) => {
+          row.querySelector('.authenticator-icon')
+            ?.setAttribute('aria-hidden', '');
```

Comment:
> What is the process with regards to changing styles/sprites for all users? It would be a somewhat significant visual change, is that something we want to push out to all users?

Not sure re; process. I'd group it with the other changes in the PR and say it's a set of noticeable visual changes that improves accessibility, i.e., bug fixes.

Reply:
For the suggested code change, it's to match the method's type signature.

```ts
// from lib.dom.d.ts
setAttribute(qualifiedName: string, value: string): void;
```

Reply:
Okay, so would it be correct then to do this instead?
```suggestion
            ?.setAttribute('aria-hidden', 'true');
```